README.md

```diff
@@ -1,13 +1,18 @@
-# Collections publisher
+# Collections Publisher
 
-The `collections-publisher` publishes certain collection and tag formats requiring
-complicated UIs.  Its initial use will be "curated lists" for browse topics.
+The Collections Publisher publishes browse pages and topic pages.
+
+The content created is served by the [Collections app](https://github.com/alphagov/collections).
+
+![Screenshot of Collections Publisher](docs/screenshot.jpg)
+
+## Nomenclature
+
+See the [README of collections frontend](https://github.com/alphagov/collections).
 
 ## Technical documentation
 
-This is a Ruby on Rails application for internal use, with no public facing
-aspect. It retrieves topic information from the Content API and publishes
-Curated lists to the Content store.
+This is a Ruby on Rails application for internal use, with no public facing aspect. It retrieves topic information from the Content API and publishes Curated lists to the Content store.
```

Comment:
"topic information" is a bit vague.  How about "information about which content is tagged to a topic or browse page"